x-pack/legacy/plugins/transform/public/shim.ts

```diff
@@ -124,6 +103,7 @@ export function createPublicShim(): { core: Core; plugins: Plugins } {
       uiMetric: {
         createUiStatsReporter,
       },
+      xsrfToken: chrome.getXsrfToken(),
```

Comment:
ML will soon be replacing our http service with kibana's core http client to remove the need of this token.